verilog/formatting/formatter.cc

```diff
@@ -84,6 +84,21 @@ static void DeterminePartitionExpansion(partition_node_type* node,
   const auto partition_policy = uwline.PartitionPolicy();
   VLOG(3) << "partition policy: " << partition_policy;
   switch (partition_policy) {
+    case PartitionPolicyEnum::kAppendFittingSubpartitions: {
+      if (verible::FitsOnLine(uwline, style)) {
```

Comment:
When using this policy, can you avoid checking for `FitsOnLine()` altogether, and just use the greedy append strategy?  You could for example, always mark this with `node_view.Expand()`, but down below format differently when reaching a partition of this type.

Reply:
I need to check if all arguments fits or not. If not then I'm expanding it all. Always marking it with `Expand()` is the same as `kAlwaysExpand`.